### PR TITLE
Internal: Extract `/core/apps` into the root directory [ED-7996]

### PR DESCRIPTION
**Pull request by udidol**

### CODE REVIEW

**Review by matipojo — CHANGES_REQUESTED** [review]
Please add BC for 3rd party plugins.
You can see some BC here:
https://github.com/matipojo/elementor/tree/b45e7fb45845f0328ebf9f44babb1933a7c42c04/core/app

File: core/app/modules/import-export/module.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+	public $import;
+
+	public function get_name() {
+		return 'import-export-bc';
+	}
+
+	public function __construct() {
+		parent::__construct();
+	}
```
Comment (matipojo) [inline core/app/modules/import-export/module.php:26]: Remove

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/app/modules/import-export/module.php (hunk 1 above)
Comment (udidol) [reply core/app/modules/import-export/module.php]: Done

File: app/assets/js/ui/grid/grid.scss [hunk 2]
```diff
@@ -0,0 +1,213 @@
+// COMPONENT: GRID
```
Comment (matipojo) [inline app/assets/js/ui/grid/grid.scss:1]: Why it's considered as a new file? It's not a move of an existing file?

File: core/app/modules/kit-library/connect/kit-library.php [hunk 3]
```diff
@@ -1,44 +1,20 @@
...
-		return $this->http_request( 'GET', "kits/{$id}/download-link" );
-	}
-
-	protected function get_api_url() {
-		return [
-			static::DEFAULT_BASE_ENDPOINT,
-			static::FALLBACK_BASE_ENDPOINT,
-		];
-	}
+	public function is_connected() {
```
Comment (matipojo) [inline core/app/modules/kit-library/connect/kit-library.php:14]: Mark the class and method as deprecated

**Review by matipojo — CHANGES_REQUESTED** [review]
Please resolve the conflicts

File: core/app/modules/kit-library/connect/kit-library.php (hunk 3 above)
Comment (udidol) [reply core/app/modules/kit-library/connect/kit-library.php:14]: WIll mark all BC classes as deprecated

Comment (udidol) [reply core/app/modules/kit-library/connect/kit-library.php:22]: Done @matipojo

**Review by matipojo — APPROVED** [review]

**Pull request merged**